oak_runtime/src/lib.rs

```diff
@@ -183,7 +189,10 @@ impl std::fmt::Debug for NodeStopper {
 }
 
 struct NodeInfo {
-    /// Name for the Node in debugging output.
+    /// The name for the node.
```

Comment:
nit: s/node/Node/ in all new/updated comments for consistency with the rest of the file.